packages/manager/modules/iam/src/components/actionSelect/actionSelect.controller.js

```diff
@@ -132,7 +132,15 @@ export default class ActionSelectController {
     this.isLoading = true;
     return this.IAMService.getActions()
       .then((actions) => {
-        this.actions = cloneDeep(actions);
+        // this.actions = cloneDeep(actions);
+        // TODO PATCH
+        // Temporary hook to manage hasQueryParams property until the API provides the value
+        this.actions = cloneDeep(actions).map((action) => ({
+          ...action,
+          hasQueryParams: action.action.endsWith('assumeRole'),
```

Comment:
**question:** Does action always endswith `assumeRule`? What if queryParams are jumbled?

Reply:
As far as I know the action always ends with `assumeRole` (it's a specific patch) but @y4nnL should confirm

Reply:
Hello,

The `hasQueryParameters` attribute should land in the `GET /iam/reference/action` API next Monday. So if we wait after the week-end, we can probably do this feature without the temporary hook.

Reply:
API team has prodded even faster than expected. It's now live world wide:

```bash
curl -X GET "https://eu.api.ovh.com/v2/iam/reference/action?resourceType=pccVMware" 

[
  {
    "action": "pccVMware:apiovh:vrops/upgrade",
    "description": "Upgrade VMware Aria Operations",
    "resourceType": "pccVMware",
    "categories": [
      "OPERATE"
    ],
    "hasQueryParameters": false
  },
  {
    "action": "pccVMware:vSphere:assumeRole",
    "description": "assume a role",
    "resourceType": "pccVMware",
    "categories": [
      "OPERATE"
    ],
    "hasQueryParameters": false
  }
]
```

Reply:
API team has been faster than expected and hasQueryParameter is now live world-wide in API.

However, there's no action with this attribute set to true yet. https://api.ovh.com/console-preview/?section=%2Fiam&branch=v2#get-/iam/reference/action

Reply:
Hello @Stoakes 
Thanks for the reactivity
Can you also please provide some actions so it can be tested ?

Reply:
For now, there's only `pccVMware:vSphere:assumeRole` which accepts query parameters. I'll try to have it online today (depends on a few reviews).

Reply:
`hasQueryParameters` set to true for `pccVMware:vSphere:assumeRole` on LABEU.